Sources/OpenSSL/ClientContext.swift

```diff
@@ -27,17 +27,16 @@ import COpenSSL
 typealias CCallback = @convention(c) Void -> Void
 
 public final class SSLClientContext: Context {
-    public init(verifyBundle: String? = nil, certificate: String? = nil, privateKey: String? = nil, certificateChain: String? = nil) throws {
+    public init(verifyBundle: String? = nil,
+      certificate: String? = nil,
+      privateKey: String? = nil,
+      certificateChain: String? = nil) throws {
 		try super.init(method: .SSLv23, type: .Client)
 
         SSL_CTX_set_verify(context, SSL_VERIFY_PEER, nil)
         SSL_CTX_set_verify_depth(context, 4)
         SSL_CTX_set_options(context, SSL_OP_NO_SSLv2 | SSL_OP_NO_SSLv3 | SSL_OP_NO_COMPRESSION)
 
-        if SSL_CTX_set_cipher_list(context, "HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4") != 1 {
-            throw Context.Error.Certificate(description: lastSSLErrorDescription)
```

Comment:
That doesn't sound good 😬 We should be kept at high security and only lower it on demand by the client, not the other way around. Can you please find out what setting we have here now, @tomohisa? I'd like to know how secure my servers are after this change.